### DISCUSSION TITLE
ErrorCode are specific to tool used in organization

### DISCUSSION THREAD

**Discussion by sahidkhangithub** [camaraproject/WorkingGroups#241] [Q&A]
Hi Team,

We are going through multiple api inorder to implement in our organization for CAMARA standard. Whereas for the error response for below errorCode  may change/depends on api tool being used for development. In general can you advice to have below error response to specific to api provider. 

ErrorCode: 401, 403, 404, 
Our tool is able to send code same as 401, 403, 404, but the fieldName and message value may change as per the tool.

**Comment by RubenBG7** [comment]
Hi  @sahidkhangithub 

It's important follow the initial purpose of CAMARA APIs persisted in the last GSMA Whitepaper: scale (developer-oriented), global reach (they are offered and aggregated by multiple operators) and simple (user-friendly semantics, easy to use and operate)

That means that we as CSPs should not throw any complex/system logic, to the CAMARA APIs.
